analysers/analyser_merge_museum_FR.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
+#-*- coding: utf-8 -*-
+
+from .Analyser_Merge import Analyser_Merge, Source, CSV, Load, Mapping, Select, Generate
+import re
+
+class Analyser_Merge_Museum_FR(Analyser_Merge):
+    def __init__(self, config, logger = None):
+
+        self.missing_official = {"item":"8010", "class": 31, "level": 3, "tag": ["merge"], "desc": T_(u"Museum not integrated") }
+        self.possible_merge   = {"item":"8011", "class": 33, "level": 3, "tag": ["merge"], "desc": T_(u"Museum, integration suggestion") }
+        self.update_official  = {"item":"8012", "class": 34, "level": 3, "tag": ["merge"], "desc": T_(u"Museum update") }
+
+        Analyser_Merge.__init__(self, config, logger,
+            u"https://www.data.gouv.fr/fr/datasets/musees-de-france-base-museofile/",
+            u"Musées de France : base Muséofile",
+            CSV(Source(attribution = u"Ministère de la Culture - Muséofile", millesime = "09/2019",
+                    fileUrl = u"https://www.data.gouv.fr/fr/datasets/r/5ccd6238-4fb0-4b2c-b14a-581909489320"),
+                separator = u';'),
+            Load("geolocalisation", "geolocalisation",
+                 where = lambda row: row["geolocalisation"],
+                 xFunction = lambda x: x and x.split(',')[1],
+                 yFunction = lambda y: y and y.split(',')[0]
+                 ),
+            Mapping(
+                select = Select(
+                    types = ["nodes", "ways"],
+                    tags = {"tourism": "museum"}),
+                conflationDistance = 50,
```

Comment:
Look, very small, it's not even the size on a museum.

Reply:
modified to 300
it was a mistake